CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 4.38.0
+### New Features
+* Paywalls: add `updateWithDisplayCloseButton` to `PaywallViewController` (#3708) via Cesar de la Vega (@vegaro)
+* New `syncAttributesAndOfferingsIfNeeded` method (#3709) via Burdock (@lburdock)
+* Add targeting to `PresentedOfferingContext` (#3730) via Josh Holtz (@joshdholtz)
+* Add `currentOffering(forPlacement: String)` to `Offerings` (#3707) via Guido Torres (@guido732)
+* New `Package.presentedOfferingContext` (#3712) via Josh Holtz (@joshdholtz)
+### Bugfixes
+*  Mark methods with StaticString for appUserID as deprecated (#3739) via Mark Villacampa (@MarkVillacampa)
+### Other Changes
+* `Xcode 15.3`: fix warnings on Integration Tests (#3603) via NachoSoto (@NachoSoto)
+* [EXTERNAL] Spelling typo fix to comment (#3713) via @vdeaugustine (#3740) via Mark Villacampa (@MarkVillacampa)
```

Comment:
This could use a manual cleanup, where we actually explain what the feature is, point to docs and how to use it

Reply:
For sure! I'll submit a PR to update these when ready 💪